### ISSUE TITLE
Conversations adding account "NEXT" to "CONNECTING" and repeat - not actually doing anything

### ISSUE THREAD

**Issue by Crasha** [siacs/Conversations#2475]
#### General information

* **Version:** Latest Release
* **Device:**  Samsung Galaxy S8
* **Android Version:** Android 6.0.1
* **Server name:** Personal (homeserver)
* **Server software:** OpenFire Latest Version

* **Installed server modules:** Default
* **Conversations source:** Latest Release (Github)

#### Steps to reproduce

1. Android 6 (although have seen it in Android 7 when I removed the account and re-added it again, but only once)
2. Latest Conversations

#### Expected result

What is the expected output? What do you see instead?

Expect to connect and login, but instead it shows "CONNECTING" then it goes back to "NEXT", no error no nothing. Works fine on Android 7, so ruled out everything and saw it was the Android Version that was only different.

Currently running Openfire on Windows, but running Openfire on Linux, Conversations seems to connect fine.

#### Debug output

Don't have any at the moment.

**Comment by iNPUTmice** [comment]
Duplicate of  #2430 and  #2439 
Probably has something to do with the Java/Windows TLS stack not offering ciphers/protocols that are available on Android 6. Both of which is out of our control.

**Comment by Crasha** [comment]
Having the same issue on Android 7 too. Just debugged on my Nexus 6P.

"xml parser mishandled SSLProtocolException(Read error: ssl=0xe1e7a340: Failure in SSL library, usually a protocol error"

**Comment by jimrome** [comment]
I have this issue on my Galaxy S8+ also. No client will connect. Get the above message or a timeout.
My old Nexus tablet on Android 6 works.
